### PR TITLE
Add support for SuSE SLES 15

### PR DESCRIPTION
**Pull request by absld**
Sucessfully tested on SuSE SLES 15.0 - no differences to SLES 12

### CODE REVIEW

**Comment by ghoneycutt** [comment]
Unrelated problem working with older versions. Attempting to solve in PR #61

**Comment by ghoneycutt** [comment]
Thanks @absld for this PR! Could you please rebase against master. Hopefully that will fix the failing tests here.

File: .travis.yml [hunk 1]
```diff
@@ -1,15 +1,15 @@
 ---
 language: ruby
 
-bundler_args: --without system_tests development
```
Comment (ghoneycutt) [inline .travis.yml]: this shouldnt be here.. i think your rebase went sideways

**Comment by ghoneycutt** [comment]
You are working from your master branch instead of a named branch. That's what is causing the problem. Using branches will ensure that you don't have issues when sending pull requests.

**Comment by ghoneycutt** [comment]
I went ahead and cherry picked the commit and made the new PR #62

**Comment by absld** [comment]
Hi, ouh, sorry for that. I'm not that familiar with using git. Thanks for the hint!
Thanks for cherry picking my changes for sles15 - fortunately there were not that much ;-)

**Pull request closed without merging**